cavy.js

```diff
@@ -55,6 +56,10 @@ program
     'Swap the index files and start the report server without first building the app'
   )
   .option('-d, --dev', 'Keep report server alive until manually killed')
+  .option(
+    '-bt, --boot-timeout <seconds>',
```

Comment:
The `--bt` option doesn't seem work - it just uses default 20 seconds. I think it has to be a single letter to work?

If you feel like a shorthand option doesn't feel right, we don't have to have one.

Reply:
Why not `-t`? 

Make sure to check none of these clash with the react-native CLI. I dunno if we checked the others though...